tools/mo/unit_tests/mo/back/ChangeOutputTypeAttributes_test.py

```diff
@@ -64,6 +64,10 @@ def test_cast_out_of_fp16_min(self):
         graph, graph_ref = build_cast_test_graphs(input_data, dst_type_str='FP16')
         self.assertRaises(Error, ChangeOutputTypeAttributes().find_and_replace_pattern, graph)
 
+    def test_cast_without_len(self):
+        input_data = np.array(4)
+        graph, graph_ref = build_cast_test_graphs(input_data, dst_type_str='FP16')
+        self.assertRaises(Error, ChangeOutputTypeAttributes().find_and_replace_pattern, graph)
```

Comment:
test is wrong. why did you pass numpy array instead of `Node` type object?

Reply:
function build_cast_test_graphs() create graphs depend on input_data. I reproduced case when input node which will be casted is scalar.

Reply:
Yes, my fault, test was incorrect. I changed it